layouts/MainLayout.tsx

```diff
@@ -1,17 +1,29 @@
 import Head from "next/head";
-import React, { ReactNode } from "react";
+import React from "react";
 
-type LayoutProps = {
-  children: ReactNode;
+type Props = {
```

Comment:
```suggestion
interface Props {
```

Not a big deal. Only change if you're here anyway.